react-elements/src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom/client';
+
+const firstReact = React.createElement(
+  'h1',
+  null,
+  'Hello, React!'
+);
+console.log(firstReact);
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+root.render();
```

Comment:
You need to pass the element to `render` so it is displayed in the browser.
```JAVASCRIPT
root.render(firstReact);
```

Reply:
gotcha! that makes sense! thanks Scott!

Reply:
![image](https://user-images.githubusercontent.com/103128961/187568858-7b0628f9-a806-49af-a58f-86070c8cf15f.png)